code/game/sound.dm

```diff
@@ -41,6 +41,8 @@
 	S.channel = channel || open_sound_channel()
 	S.volume = vol
 	S.environment = 7
+	if(stream)
+		S.status = SOUND_STREAM
```

Comment:
Heard there are some issues with the TM. I suppose it could be the song sound status flags being overriden.
```suggestion
		S.status |= SOUND_STREAM
```

Reply:
Done